scripts/calculate_regressions.py

```diff
@@ -0,0 +1,72 @@
+import argparse
+import dataclasses
+import json
+from dataclasses import dataclass
+from typing import List, Iterable, Optional
+
+
+@dataclass(frozen=True)
+class Annotation(object):
+    filePath: str
+    line: int
+    column: int
+    highlightedText: str
+    error: str
+    inspectionToolId: Optional[str]
+
+    def __str__(self) -> str:
+        if self.inspectionToolId is not None:
+            suffix = f" by {self.inspectionToolId}"
+        else:
+            suffix = ""
+        return f"{self.filePath}:{self.line}:{self.column} '{self.highlightedText}' ({self.error}){suffix}"
+
+    @staticmethod
+    def from_dict(raw_dict):
+        return Annotation(raw_dict["filePath"],
+                          raw_dict["line"],
+                          raw_dict["column"],
+                          raw_dict["highlightedText"],
+                          raw_dict["error"],
+                          raw_dict["inspectionToolId"])
+
+
+def read_data(path: str) -> List[Annotation]:
+    with open(path) as json_file:
+        data = json.load(json_file)
+
+    return list(map(lambda value: Annotation.from_dict(value), data))
```

Comment:
List comprehension is such much readable in my opinion 😄

```python
[Annotation.from_dict(value) for value in data]
```

Reply:
Cool!
It would be great to have an intention in python plugin to make such conversion automatically